vectara_agentic/llm_utils.py

```diff
@@ -136,9 +138,22 @@ def get_llm(role: LLMRole, config: Optional[AgentConfig] = None) -> LLM:
 
         llm = Fireworks(model=model_name, temperature=0, max_tokens=max_tokens)
     elif model_provider == ModelProvider.BEDROCK:
-        from llama_index.llms.bedrock import Bedrock
+        from llama_index.llms.bedrock_converse import BedrockConverse
+
+        aws_profile_name = os.getenv("AWS_PROFILE", None)
```

Comment:
I think this should be optional, profile_name / region_name are optional on bedrockconverse and ideally it should follow the normal aws cred chain if not specified.